### DISCUSSION TITLE
Updated light-teal.css file for improved visibility of text

### DISCUSSION THREAD

**Discussion by CalculonPrime** [vladmandic/sdnext#3606] [General]
Looks like if I want to edit the file from your repo to suggest changes, I'd have to fork the whole thing first.

So just attaching an improved version of light-teal.css; this makes the text in the model table cells visible and improves the visibility of text in picker dropdowns as well.
[light-teal.zip](https://github.com/user-attachments/files/17966065/light-teal.zip)

**Comment by vladmandic** [comment]
i totally understand that creating a fork just to do couple of css edits is overkill - thus this is ok.
working on a sidebranch now, as soon as i'm back on dev i'll merge this.

**Reply by vladmandic** [reply]
its now merged to dev branch.
